src/Jackett.Common/Indexers/NCore.cs

```diff
@@ -272,6 +272,13 @@ protected async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query,
             CQ dom = results.Content;
             int numVal = 0;
 
+            // find number of torrents / page
+            int torrent_per_page = dom[".box_torrent_all"].Find(".box_torrent").Length;
+            int start_page = (query.Offset / torrent_per_page)+1;
```

Comment:
DivideByZeroException if there are no torrents on the page. (nothing found)

Reply:
Resolve with #10397a17afb4ef3fb3ae5e9f6d303712cd12fddd